src/cmds/scripts/post_install.py

```diff
@@ -0,0 +1,463 @@
+print "Post install process started..."
```

Comment:
Doesn't the new postinstall script for Windows constitute a new interface? Shouldn't there be a design document to accompany the changes?